examples/getting_started/plot_skore_getting_started.py

```diff
@@ -34,38 +34,13 @@
 # ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 # %%
-# .. note::
-#
-#   If we do not wish for our skore project to be stored in a *temporary* folder, we
-#   can simply create and load the project in the current directory with:
-#
-#   .. code-block:: python
-#
-#     import skore
-#
-#     my_project = skore.open("my_project")
-#
-#   This would create a skore project directory named ``my_project.skore`` in our
+#   Let's start by creating a skore project directory named ``my_project.skore`` in our
 #   current directory.
 
-# %%
-# Here, we start by creating a temporary directory to store our project so that we can
-# easily clean it after executing this example:
-
-# %%
-import tempfile
-from pathlib import Path
-
-temp_dir = tempfile.TemporaryDirectory(prefix="skore_example_")
-temp_dir_path = Path(temp_dir.name)
-
-# %%
-# Then, we create and load the skore project from this temporary directory:
-
 # %%
 import skore
 
-my_project = skore.open(temp_dir_path / "my_project")
+my_project = skore.open("my_project", create=True)
```

Comment:
By default, `create=True`
```suggestion
my_project = skore.open("my_project")
```

Reply:
it was also a remark from Auguste, and here is my answer:  
![image](https://github.com/user-attachments/assets/0c034b5e-3a2e-494d-8b60-3d028d488c03)  

Do you still want me to remove `create=True`?